### PR TITLE
Refactor histograms into FacetViews

### PR DESCRIPTION
**Pull request by brianhelba**

### CODE REVIEW

**Comment by brianhelba** [comment]
@ronichoudhury PTAL. We can take a look at this together if you'd like an outline of the changes.

File: web_external/js/views/body/ImagesView/ImagesFacetsPane.js [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+            }
+
+            var facetView = new FacetView({
+                model: this.model,
+                facetName: facetName,
+                parentView: this
+            });
+
+            this.facetViews[facetName] = facetView;
+            this.$el.append(facetView.el);
```
Comment (brianhelba) [inline web_external/js/views/body/ImagesView/ImagesFacetsPane.js]: Hey @msmolens , I found it useful here to let Backbone create the `el` for each `FacetView` from scratch, then append that new element into the existing main DOM. I have the feeling that we should probably take better advantage of this pattern in other parts of the code.

Comment (msmolens) [inline web_external/js/views/body/ImagesView/ImagesFacetsPane.js]: For dynamic, repeated elements we typically use iteration in the jade templates and avoid explicit code like `this.$el.append()`, but I agree we should keep this style in mind.

File: web_external/js/views/body/ImagesView/ImagesFacetsPane.js [hunk 2]
```diff
@@ -0,0 +1,48 @@
+isic.views.ImagesFacetsPane = isic.View.extend({
+    render: function () {
+        if (!this.addedCollapseImage) {
```
Comment (ronichoudhury) [inline web_external/js/views/body/ImagesView/ImagesFacetsPane.js]: I agree. We kept it around in this PR just out of caution, and resolved to cut out unneeded assets and associated code in a separate sweep.

**Pull request merged**